frontend/src/pages/Account/components/OrderPreview/OrderPreview.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC } from "react";
+import { Link } from "react-router-dom";
+import { OrderProps } from "./types";
+import ProductCard from "@/components/ProductCard/ProductCard";
+
+const formatDate = (date: Date) => {
+  const dayOfWeek = date.toLocaleDateString("en-US", { weekday: "long" });
+  const day = date.toLocaleDateString("en-US", { day: "2-digit" });
+  const month = date.toLocaleDateString("en-US", { month: "short" });
+  return `${dayOfWeek} ${day} ${month}`;
+};
```

Comment:
These functions are identical. It would be better to extract them into a separate file to avoid duplication.